e2e-tests/cross-site-sharded/run

```diff
@@ -13,6 +13,25 @@ unset OPERATOR_NS
 main_cluster="cross-site-sharded-main"
 replica_cluster="cross-site-sharded-replica"
 
+wait_for_members() {
+  local endpoint="$1"
+  nodes_amount=0
+  until [[ ${nodes_amount} == 6 ]]; do
+    nodes_amount=$(run_mongos 'rs.conf().members.length' "clusterAdmin:clusterAdmin123456@$endpoint" "mongodb" ":27017" \
+     | egrep -v 'I NETWORK|W NETWORK|Error saving history file|Percona Server for MongoDB|connecting to:|Unable to reach primary for set|Implicit session:|versions do not match|Error saving history file:|bye' \
+     | $sed -re 's/ObjectId\("[0-9a-f]+"\)//; s/-[0-9]+.svc/-xxx.svc/')
+
+    echo "$nodes_amount"
```

Comment:
it would be better to have more descriptive log like: "${nodes_amount} members is in replset ${rsName} configuration"

Reply:
and it'd be good to print `waiting for all members to be configured in ${replsetName}` before `until`